src/app/common/schemas.py

```diff
@@ -125,28 +185,112 @@ def load_from_fitz_rect(rect: fitz.Rect) -> "BoundingBox":
 
 
 class Coordinates(BaseModel):
-    """Coordinates schema."""
+    """Coordinates schema for representing geographical data points.
 
-    east: float = Field(..., example=1.0)
-    north: float = Field(..., example=2.0)
-    projection: str = Field(..., example="LV95")
+    This schema defines the format for specifying location data using east/north coordinates
+    along with the projection system used for geo-referencing.
```

Comment:
```suggestion
    along with the projection system used.
```
Let's remove the "for geo-referencing" as this is something slightly different to my understanding (e.g. aligning a satellite image with a fixed coordinate system, whereas here we are just reading some specific coordinates, not aligning different coordinate systems).

Reply:
I see. Sorry, I was not aware of this difference! Good lesson learned for today!